### PR TITLE
Update eigenvalue page

### PR DESCRIPTION
**Pull request by pascaladhikary**
- reorganize in slide order
- include finding eval/evec example, code snippet
- replace cost summary with cost/convergence table

### CODE REVIEW

File: notes/eigen.md [hunk 1]
```diff
@@ -285,19 +309,30 @@ $$ \mathbf{e}_{k+1} \approx \frac{|\lambda_2|}{|\lambda_1|}\mathbf{e}_k$$.
...
 $$ \mathbf{e}_{k+1} \approx \frac{|\lambda_\text{closest} - \sigma|}{|\lambda_\text{second-closest} - \sigma|}\mathbf{e}_k.$$
 
-## Cost Summary
+## Cost and Convergence Summary
+
+| Method                         | Description                                                                             | Cost             | Convergence                                           |
+|--------------------------------|-----------------------------------------------------------------------------------------|------------------|-------------------------------------------------------|
+| Power Method                   | $$\boldsymbol{x}_{k+1} = \mathbf{A} \boldsymbol{x}_{k}$$                               | $$kn^2$$         | $$\left\|\frac{\lambda_2}{\lambda_1}\right\|$$       |
+| Inverse Power Method           | $$\mathbf{A} \boldsymbol{x}_{k+1} = \boldsymbol{x}_{k}$$                                | $$n^{3} + kn^2$$ | $$\left\|\frac{\lambda_n}{\lambda_{n-1}}\right\|$$   |
+| Shifted Inverse Power Method   | $$(\mathbf{A} - \sigma \mathbf{I}) \boldsymbol{x}_{k+1} = \boldsymbol{x}_{k}$$          | $$n^{3} + kn^2$$ | $$\left\|\frac{\lambda_c-\sigma}{\lambda_{c2}-\sigma}\right\|$$ |
```
Comment (jlevine18) [inline notes/eigen.md:319]: maybe add a bit more explanation of runtime? especially with lu factorization

**Review by jlevine18 — CHANGES_REQUESTED** [review]
this is a good start- see my comments. i'd also take a bit of a closer look at some of the power iteration section that we currently have and give a little more to help students build intuition- we ask a lot of questions about errors/convergence/advantage of normalization, which the notes are somewhat lacking in. i'd also add a couple graphical examples as well to show eigenvals/eigenvecs as deformation directions, and convergence during the steps of power iteration. i'd also look into moving some stuff around in the power-iteration section and making that the whole text is cohesive and reads well. feel free to do things a little differently- the most important goals are that the material contains everything someone might need to do hw/a quiz and that it reads well

**Comment by jlevine18** [comment]
also it's probably worth doing one last check to make sure that everything from the slides/videos is in here

**Review by jlevine18 — APPROVED** [review]
looks good- thanks for your help

**Review by devksingh4 — COMMENTED** [review]
lgtm

**Pull request merged**